Add validation to pack name is changed (#1800)
demisto/demisto-sdk: CHANGELOG.md

* Added support for beta integrations in the **format** command.
* Fixed an issue where the **postman-codegen** command failed when called with the `--config-out` flag.
* Removed the integration documentation from the detailed description while performing **split** command to the unified yml file.

> * Add to **validate** a validation for pack name to make sure it is unchanged.

Reviewer: ```suggestion
* Added a test to **validate**, to make sure sure pack paths are not changed.
```
